src/modules/hivemq/hivemq-container.ts

```diff
@@ -0,0 +1,56 @@
+import * as path from "path";
+
+import { GenericContainer } from "../../generic-container/generic-container";
+import { StartedTestContainer } from "../../test-container";
+import { AbstractStartedContainer } from "../abstract-started-container";
+import { Wait } from "../../wait-strategy/wait";
+
+const START_LOG_MESSAGE_REGEX = /(.*)Started HiveMQ in(.*)/i;
+const HIVEMQ_BASE_PATH = "/opt/hivemq";
+const MQTT_PORT = 1883;
+
+export enum HiveMQDockerImageName {
+  COMMUNITY_EDITION = "hivemq/hivemq-ce",
+}
+
+export class HiveMQContainer extends GenericContainer {
+  private temporaryFileSystemOptions = new Map<string, string>();
+
+  constructor() {
+    super(HiveMQDockerImageName.COMMUNITY_EDITION);
```

Comment:
I would add a default image to the constructor like here. In this way we can drop the enum: https://github.com/testcontainers/testcontainers-node/blob/9f0b3afb6e361a0e87c9873e7417cb9d8f5058ef/src/modules/postgresql/postgresql-container.ts#L13

Reply:
fixed version 👍